docs/contributors/opentracingprovider.adoc

```diff
@@ -188,4 +189,34 @@ When the ‘inject’ method is called on the OpenTracing Tracer, it will provid
 An unique id should be created, associated with the trace state being returned for injection into the outbound message, and used to create an `Interaction` based correlation id for the current node. The node should also be created as a `Producer`.
 
 
+=== Sampling 
 
+Sampling implementation is not required, however if a provider decideds to implement it should follow following rules.
```

Comment:
This section talks about implementing opentracing provider right? 

What I wanted to stress is that is somebody implements a provider without sampling then the provider should set reporting level to 'All' so by default all providers should implement some kind of `AlwaysSampler` I will update the doc